validation/data-availability-type_test.go

```diff
@@ -20,10 +20,7 @@ import (
 
 func testDataAvailabilityType(t *testing.T, chain *ChainConfig) {
 	rpcEndpoint := Superchains[chain.Superchain].Config.L1.PublicRPC
-	require.NotEmpty(t, rpcEndpoint)
-
-	client, err := ethclient.Dial(rpcEndpoint)
-	require.NoErrorf(t, err, "could not dial rpc endpoint %s", rpcEndpoint)
```

Comment:
in theory since this condition was already checked at the start of the test suite run, it shouldn't fail here either besides in edge-case network conditions. do you think that warrants keeping them in? i'm fine reverting if so, just didn't see much value in that given the small time difference between them

Reply:
Yea I think if we want to eliminate these redundant connection error checks, we should just make a single ethclient connection and pass the client to each underlying dependent. We are working towards that in some of the refactors discussed [here](https://github.com/ethereum-optimism/superchain-registry/pull/802#discussion_r1889403017). I'd say revert the error check removal for now and we'll get rid of the redundancy there.